scripts/manage_porkbun.py

```
@@ -63,137 +340,415 @@ def init_config(config_location="../etc/config.ini"):
-            i["type"] == "A" or i["type"] == "ALIAS" or i["type"] == "CNAME"
-        ):
-            print("Deleting existing " + i["type"] + " Record")
-            deleteRecord = json.loads(
-                requests.post(
-                    apiConfig["endpoint"] + "/dns/delete/" + rootDomain + "/" + i["id"],
-                    data=json.dumps(apiConfig),
-                ).text
-            )
+    try:
+        payload = json.dumps(req_data)
+        req = requests.post(url, data=payload)
```

Ruff (0.15.21)

[error] Probable use of `requests` call without timeout

(S113)